package.json

```diff
@@ -43,11 +43,13 @@
     "test:playwright:debug": "PWDEBUG=1 pnpm test:playwright:local",
     "test:playwright:recreate-tapes": "rimraf test/tapes && pnpm test:playwright:update-tapes",
     "test:playwright:update-tapes": "UPDATE_TAPES=true pnpm test:playwright",
+    "test:playwright:update-snapshots": "pnpm test:playwright visual-regression --update-snapshots",
```

Comment:
There is some documentation in `test/playwright/README.md` about using the `-u` flag that needs to be updated to refer to this script instead.